pkg/gardenlet/controller/seed/seed/add.go

```diff
@@ -77,6 +78,7 @@ func (r *Reconciler) AddToManager(mgr manager.Manager, gardenCluster cluster.Clu
 	return c.Watch(
 		source.NewKindWithCache(&gardencorev1beta1.Seed{}, gardenCluster.GetCache()),
 		&handler.EnqueueRequestForObject{},
+		predicateutils.HasName(r.Config.SeedConfig.Name),
```

Comment:
Can you explain what we did here? The predicate is checking if the `Seed` object has the same name as the name in the SeedConfig of the gardenlet, right? How does this help to access the other seed?

Reply:
The cache is no longer centrally restricted in `app.go`, i.e. the gardenlet will watch all `Seed`s/have them in its cache. With this change, the controller has to make sure to only reconcile the `Seed` object it is responsible for. This is achieved with this predicate (basically the same like this `SeedFilterFunc` in `master` branch).